assets/js/components/Dashboard/PaMessageForm/MainForm.tsx

```diff
@@ -302,10 +306,21 @@ const MainForm = ({
                       <Button
                         className={paMessageStyles.serviceTimeButton}
                         variant="link"
-                        onClick={() => setEndTime("03:00")}
+                        onClick={() => {
+                          if (moment(endTime, "HH:mm").hour() >= 3) {
```

Comment:
Hmm, good question. Because we use time and date for input type, I'm not sure the user can get into that situation. I will still add a `moment().isValid()` check to prevent any weird behavior.

Reply:
Actually, I forgot there are other ways it can be invalid. I wonder if I should just extract `isValid` from the time picker into a variable and `return` early if that is false. This may have been the original idea behind your question so apologies if I'm overthinking this.